dialdbg/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
Just copied the license from the top level rust-utils. Let me know if we want to do something different for `dialdbg`.